src/createBlock.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * Creates the next block in the Raha Blockchain from unapplied operations in Firestore.
+ *
+ * Example usage:
+ * node dist/createBlock.js > newBlock.json
+ */
+
+import { getBlockchain } from './RahaBlockchain';
+import { operationsCollectionFilters as filters, get, operationsCollection } from './RahaFirestore';
+import { BLOCKCHAIN_VERSION_NO, Block, StellarMetadata, IpfsBlock, Operation, RequestInvite, Trust } from './schema';
+
+/**
+ * A map of functions that build the data component of Blockchain Operations from
+ * the data component of Firestore Operations.
+ */
+const bcOpDataBuilders = {
```

Comment:
Can we just generally not abbreviate all this stuff? It makes it look arcane to newbies without actually achieving anything.